VM Translation to It (#459)
cncf/glossary: content/it/virtual-machine.md

---
title: Macchina virtuale (VM)
status: Feedback Appreciated 

> category: Technology

Reviewer: ```suggestion
category: Tecnologia
```

---

## Che cos’è

> Una macchina virtuale (VM) è un computer e il suo sistema operativo che non è legato a un particolare componente hardware. Le macchine virtuali si basano sulla [virtualizzazione] (/virtualization/) per suddividere un singolo computer fisico in più computer virtuali. Questa separazione consente alle organizzazioni e ai fornitori di infrastrutture di creare e distruggere VM senza influire sull'hardware sottostante.

Reviewer: ```suggestion
Una macchina virtuale (VM) è un computer e il suo sistema operativo che non è legato a un particolare componente hardware. Le macchine virtuali si basano sulla [virtualizzazione](/virtualization/) per suddividere un singolo computer fisico in più computer virtuali. Questa separazione consente alle organizzazioni e ai fornitori di infrastrutture di creare e distruggere VM senza influire sull'hardware sottostante.
```

## Quale problema affronta

> Le macchine virtuali sfruttano la virtualizzazione. Quando una macchina di metallo [bare metal] (/bare_metal_machine/) è vincolata a un singolo sistema operativo (OS), il modo in cui le risorse della macchina possono essere utilizzate è alquanto limitato. Inoltre, quando un sistema operativo è legato a una singola macchina fisica, la sua disponibilità è direttamente legata a quell'hardware. Se la macchina fisica è offline a causa di manutenzione o guasti hardware, lo è anche il sistema operativo.

Reviewer: mi pare abbiamo deciso in qualche altra PR di utilizzare il termine bare metal, senza traduzioni
```suggestion
Le macchine virtuali sfruttano la virtualizzazione. Quando una macchina [bare metal](/bare_metal_machine/) è vincolata a un singolo sistema operativo (OS), il modo in cui le risorse della macchina possono essere utilizzate è alquanto limitato. Inoltre, quando un sistema operativo è legato a una singola macchina fisica, la sua disponibilità è direttamente legata a quell'hardware. Se la macchina fisica è offline a causa di manutenzione o guasti hardware, lo è anche il sistema operativo.
```

## In che modo aiuta

> Rimuovendo la relazione diretta tra un sistema operativo e una singola macchina fisica, si risolvono diversi problemi delle macchine bare metal: tempo di approvigionamento (provisioning), utilizzo dell'hardware e resilienza.

Reviewer: ```suggestion
Rimuovendo la relazione diretta tra un sistema operativo e una singola macchina fisica, si risolvono diversi problemi delle macchine bare metal: tempo di provisioning (approvvigionamento), utilizzo dell'hardware e resilienza.
```

> I tempi di provisioning per un nuovo computer vengono notevolmente migliorati, senza l'acquisto, l'installazione o la configurazione di nuovo hardware per supportarlo. Le macchine virtuali consentono di utilizzare meglio le risorse hardware fisiche esistenti posizionando più macchine virtuali su una singola macchina fisica. Non vincolate a una macchina fisica particolare, le macchine virtuali sono anche più resilienti delle macchine fisiche. Quando una macchina fisica deve andare offline, le macchine virtuali in esecuzione su di essa possono essere spostate su un'altra macchina con tempi di inattività minimi o nulli

Reviewer: ```suggestion
I tempi di provisioning per un nuovo computer vengono notevolmente migliorati, senza l'acquisto, l'installazione o la configurazione di nuovo hardware per supportarlo. Le macchine virtuali consentono di utilizzare meglio le risorse hardware fisiche esistenti, creando più macchine virtuali su una singola macchina fisica. Non vincolate a una macchina fisica particolare, le macchine virtuali sono anche più resilienti delle macchine fisiche. Quando una macchina fisica deve essere spenta, le macchine virtuali in esecuzione su di essa possono essere spostate su un'altra macchina con tempi di inattività minimi o nulli.
```

---
title: Macchina virtuale (VM)
status: Completed

> category: Technologia

Reviewer: ```suggestion
category: Tecnologia
```
